src/components/OrderBookChart.stories.tsx

```diff
@@ -0,0 +1,256 @@
+    data: { control: null },
+  },
+  decorators: [(Story): JSX.Element => <div style={{ height: '97vh' }}>{Story()}</div>],
+} as Meta
+
+//-------------------REAL-DATA------------------
+
+// Template for real data
+// fetched from price-estimation service
+const RealTemplate: Story<OrderBookChartProps> = (args) => <OrderBookChart {...args} />
+
+const DAI = tokenList.find((token) => token.symbol === 'DAI')
+const USDC = tokenList.find((token) => token.symbol === 'USDC')
+export const USDC_DAI = RealTemplate.bind({})
+USDC_DAI.args = {
+  ...defaultParams,
+  baseToken: DAI || tokenList[0],
+  quoteToken: USDC || tokenList[1],
+  data: processRawApiData({ data: realData, baseToken: DAI || tokenList[0], quoteToken: USDC || tokenList[1] }),
+}
+USDC_DAI.argTypes = {
+  baseToken: { control: null },
+  quoteToken: { control: null },
+  data: { control: null },
+}
+
+//-------------------SAMPLE-DATA------------------
+
+interface ProduceOrderBookArgsParams {
+  sampleData: Pick<SampleData, 'asks' | 'bids'>
+  orderbookProps: Omit<OrderBookChartProps, 'data'>
+}
+// normalizing sample data
+// because real data is slightly more complex
+const produceOrderBookProps = ({ sampleData, orderbookProps }: ProduceOrderBookArgsParams): OrderBookChartProps => {
+  const { baseToken, quoteToken } = orderbookProps
+
+  const { asks = [], bids = [] } = sampleData
+  const normalizedRawData: RawApiData = {
+    asks: asks.map(({ amount, price }) => ({
+      // turn small numbers into wei or decimal-aware units
+      // otherwise get filtered as too small by src/components/OrderBookWidget.tsx#L77
+      volume: amount * 10 ** baseToken.decimals,
+      price,
+    })),
+    bids: bids.map(({ amount, price }) => ({
+      volume: amount * 10 ** quoteToken.decimals,
+      price,
+    })),
+  }
+
+  const processedData = processRawApiData({ data: normalizedRawData, baseToken, quoteToken })
+
+  return { ...orderbookProps, data: processedData }
+}
+
+// data samples
+const [
+  liquidMarket,
+  liquidMarketBigSpread,
+  liquidMarketWhenPricesFall20,
+  liquidMarketWhenPricesFall40,
+  liquidMarketWhenPricesFall60,
+  lowVolumeOverlap,
+  highVolumeOverlap,
+  lowVolumeOverlapWithBigPriceDifference,
+  highVolumeOverlapWithBigPriceDifference,
+  spreadIsAlmostNonExistent,
+  bidsLiquidAsksIlliquid,
+  asksLiquidBidsIlliquid,
+  asksLiquidInTheEdgesBidsIlliquid,
+  bidsLiquidInTheEdgesAsksIlliquid,
+  noBids,
+  noAsks,
+  noBidsAndNoAsks,
+] = sampleDataSet
+
+type SampleProps = SampleData & {
+  baseToken: string
+  quoteToken: string
+  networkId: number
+}
+
+// preprocessing template
+// does what OrderBookWidget does for real data
+const SampleTemplate: Story<SampleProps> = ({
+  name,
+  description,
+  asks,
+  bids,
+  networkId,
+  baseToken: baseTokenSymbol,
+  quoteToken: quoteTokenSymbol,
+}) => {
+  const baseToken = useMemo(() => {
+    return (
+      tokenList.find((token) => token.symbol === baseTokenSymbol || token.address === baseTokenSymbol) ||
```

Comment:
Can't you use the token find method from utils?

Reply:
They are not equivalent, `getToken` from utils doesn't have a fallback like `symbol || address`,not without starting another `.find`